### ISSUE TITLE
Option to list group projects

### ISSUE THREAD

**Issue by erikthered** [makkes/gitlab-cli#19]
The `projects` command only shows personal projects and I couldn't see a way to view projects from groups you're a member of. It would be nice to have a way to view these from the CLI.

**Comment by erikthered** [comment]
I can work on a PR for this if you're open to it

**Comment by makkes** [comment]
Awesome! Any questions, I'm here to help out.

**Comment by erikthered** [comment]
Looking over the API, gitlab-cli is currently using the "User Projects" api: https://docs.gitlab.com/ee/api/projects.html#list-user-projects

I think this feature would likely also leverage the "All Projects" api: https://docs.gitlab.com/ee/api/projects.html#list-all-projects

At this point I'm just trying to think through how to add functionality without making it unwieldy. The /projects API shows _all_ public projects by default which isn't too useful. However, using the `owned` and/or `membership` flags should accomplish what I described in the issue. Pagination #21 could quickly come into play here as well.

Just spitballing here, but the default behavior for `projects` could stay the same, and then maybe I could add `--owned` and `--membership` to toggle to the "All Projects" API with the relevant flags.

**Comment by makkes** [comment]
Sounds great. I like the idea of using flags to toggle between contexts. Do you think pagination should be done beforehand to not make adding it afterwards harder? Otherwise I'd say go for it.

**Comment by erikthered** [comment]
It probably does make sense to do pagination before this feature, since it's a cross-cutting concern and will impact most commands. I'll read up on pagination and think about an approach, I'll update #21 with what I'm thinking.

**Comment by erikthered** [comment]
I was able to get a `--member` flag working pretty quick to show projects you're a member of (basically `/projects?membership=true`. However, it seems like a sort option might be a good add-on to this, I think they're sorted by project id by default which isn't terribly intuitive.
